moped-editor/src/views/projects/projectView/ProjectWorkActivity/ProjectWorkActivityTable.js

```diff
@@ -55,33 +57,45 @@ const useColumns = ({ deleteInProgress, onDeleteActivity, setEditActivity }) =>
         headerName: "ID",
         field: "reference_id",
         minWidth: 125,
+        flex: 1,
```

Comment:
setting a `minWidth` and ` flex` prop seems to be winning combo for making the column layouts bearable. The columns will expand to fill the whole width of the table, unless they don't all fit, in which case they shrink down to their min width and the table side-scrolls.

Reply:
That feels right to me imo and eliminates the decisions around what should take up how much space which is nice.